oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -305,10 +307,15 @@ Trigger events for removing a CC would be:
 For those wishing to indicate they are leaving the CC program, or to report a
 Code of Conduct violation, please `Contact the Program Administrators`_.
 
-Program administrators will be responsible for determining when a CC is
-inactive/unreachable, by doing a check at least once per quarter. Since the
-reasoning for removing inactive CCs is due to security concerns, those who've
-been removed for inactivity may get in touch with Program Administrators to
+Core contributors are responsible for reporting when they aren't able to be
+sufficiently available to fulfill their commitments. Program administrators
```

Comment:
It's enough to say, "Core contributors are responsible for reporting when they aren't able to fulfill their commitments."  One of the commitments is being available.